File: content/snippet/code-signal/arcade/intro/diving-deeper/extracteach-kth/extracteach-kth.js
function extractEachKth(input_list, k) {
  return input_list.filter((_, i) => (i + 1) % k !== 0);
}

console.log(extractEachKth([1, 2, 3, 4, 5, 6, 7, 8, 9, 10], 3));

File: docs/snippet/code-signal/arcade/intro/diving-deeper/extracteach-kth/extracteach-kth.py
def extracteach_kth(input_list, k):
    return [input_list[i] for i in range(len(input_list)) if i % k != k - 1]


print(extracteach_kth([1, 2, 3, 4, 5, 6, 7, 8, 9, 10], 3))
